Examples/xc.helloworld/build.cake

```diff
@@ -123,6 +124,29 @@ Task("GenerateRuntimeCmd")
     fileContents += "start runSpy.cmd\n";
     FileWriteText(@"run.cmd", fileContents);
 });
+
+Task("GenerateClusteredRuntimeCmd")
+  .Does(() => {
+    var fileContents = "";
+    foreach(var xcrFile in GetFiles("./Runtime/xcassemblies/*.xcr"))
+    {
+      var xcPropertiesPath = xcrFile.FullPath.Replace("xcr", "xcproperties");
+      var xcRuntimeBinaryFilePath = MakeAbsolute(File(toolsRoot + @"XCBuild/XCRuntime/xcruntime.exe"));
+      var runServiceCmd = "start " + xcRuntimeBinaryFilePath + " " + xcrFile.FullPath + " " + xcPropertiesPath + "\n";
```

Comment:
1. you need to start it in cluster mode
2. you need to start more than one instance